### PR TITLE
SALTO-2105: Netsuite - throw on accountId that contains whitespaces

### PR DESCRIPTION
**Pull request by omrilit**
_Release Notes_: 
None

### CODE REVIEW

File: packages/netsuite-adapter/src/adapter_creator.ts [hunk 1]
```diff
@@ -201,6 +201,13 @@ const throwOnMissingSuiteAppLoginCreds = (credentials: Credentials): void => {
 const netsuiteCredentialsFromCredentials = (
   credsInstance: Readonly<InstanceElement>
 ): Credentials => {
+  const throwOnInvalidAccountId = (credentials: Credentials): void => {
+    const accountIdHasWhitespaceChar = /\s/g.test(credentials.accountId)
+    if (credentials.accountId.length === 0 || accountIdHasWhitespaceChar) {
+      throw Error(`received an invalid accountId value: (${credsInstance.value.accountId}). The accountId should not be empty nor contain whitespaces`)
```
Comment (ori-moisis) [inline packages/netsuite-adapter/src/adapter_creator.ts:207]: @amitgoldarb - note that the more we delay fixing the interface for validate credentials, the more code we will need to change, like this new addition that definitely should not throw an exception but I guess this is what we are doing now...

Comment (omrilit) [reply packages/netsuite-adapter/src/adapter_creator.ts:207]: @ori-moisis I guess this planned interface refactor shouldn't block this PR right?

Comment (ori-moisis) [reply packages/netsuite-adapter/src/adapter_creator.ts:207]: yes, just letting @amitgoldarb know about it.

**Review by dantal4 — APPROVED** [review]
lg

**Pull request merged**